docs/baking.md

```diff
@@ -159,71 +137,97 @@ the key:
 
 In order to activate account run:
 ```
-sudo -u tezos tezos-client activate account <alias> <path-to-downloaded-json>
+sudo -u tezos tezos-client activate account baker with <path-to-downloaded-json>
 ```
 
 2) You know either the unencrypted or password-encrypted secret key for your address.
 
 In order to import such a key, run:
 ```
-sudo -u tezos tezos-client import secret key <alias> <secret-key>
+sudo -u tezos tezos-client import secret key baker <secret-key>
 ```
 3) The secret key is stored on a ledger.
 
-Open the Tezoz Wallet app on your ledger and run the following
+Open the Tezos Wallet app on your ledger and run the following
 to import the key:
 ```
-sudo -u tezos tezos-client import secret key <alias> <ledger-url>
+sudo -u tezos tezos-client import secret key baker <ledger-url>
 ```
 Apart from importing the key, you'll also need to set it up for baking. Open Tezos Baking app
 on your ledger and run the following:
 ```
-sudo -u tezos tezos-client setup ledger to bake for <alias>
+sudo -u tezos tezos-client setup ledger to bake for baker
 ```
 
 ### Registering the baker
 Once the key is imported, you'll need to register your baker, in order to do that run the following
 command:
 ```
-sudo -u tezos tezos-client register key <alias> as delegate
+sudo -u tezos tezos-client register key baker as delegate
 ```
 
 Check a blockchain explorer (e.g. https://tzkt.io/ or https://tzstats.com/) to see the baker status and
 baking rights of your account.
 
-### Updating baker and endorser daemons configuration
-Apart from that, you'll need to update the `BAKER_ACCOUNT` and `ENDORSER_ACCOUNT` (in
-`/etc/default/tezos-baker-<proto>` and `/etc/default/tezos-endorser-<proto>` respectively) in
-accordance to the **alias** of the imported key.
+### Using different baking account alias
+
+By default `tezos-baking-<network>.service` will use the `baker` alias for the
+key that will be used for baking and endorsing.
+
+In case you want to use different alias for the baking account, follow two previous sections
+and replace `baker` with the desired alias. Also, you'll need to update `BAKER_ADDRESS_ALIAS`
+in `/etc/default/tezos-baking-<network>`.
```

Comment:
:thinking: I though I made this comment before, but maybe it kinda got lost between the other changes.

I think this should be right before the `Importing the baker key` section above.

IMO if one want to change the alias he'll want to do so before reading the instructions to import the key.

Reply:
makes sense